charts/apps/notebook-controller/values.yaml

```diff
@@ -1,2 +1,3 @@
-null
-...
+cullIdleTime: 30
```

Comment:
update this to

```
cullingPolicy:
  enableCulling:
  cullIdleTime:
  idlenessCheckPeriod:
```

and make corresponding chart changes

Reply:
done